### ISSUE TITLE
Does this React component not support SSO feature?

### ISSUE THREAD

**Issue by jaequery** [disqus/disqus-react#135]
### Is your feature request related to a problem? Please describe.  
Unable to pass SSO data to the component

### Describe the solution you'd like  
Need to be able to pass remote_auth_s3 to the configuration option

### Describe alternatives you've considered  
Looked at a previous PR (https://github.com/disqus/disqus-react/pull/17) that says it's now added but when I try, I get a TS error saying remoteAuthS3 (also tried remote_auth_s3) is not supported in the DisqusConfig.

**Comment by jaequery** [comment]
here is the error in detail:

```
No overload matches this call.
  Overload 1 of 2, '(props: DiscussionEmbedProps | Readonly<DiscussionEmbedProps>): DiscussionEmbed', gave the following error.
    Type '{ url: string; identifier: string; title: string | undefined; language: string; remote_auth_s3: string; }' is not assignable to type 'DiscussionEmbedConfig'.
      Object literal may only specify known properties, and 'remote_auth_s3' does not exist in type 'DiscussionEmbedConfig'.
  Overload 2 of 2, '(props: DiscussionEmbedProps, context: any): DiscussionEmbed', gave the following error.
    Type '{ url: string; identifier: string; title: string | undefined; language: string; remote_auth_s3: string; }' is not assignable to type 'DiscussionEmbedConfig'.
      Object literal may only specify known properties, and 'remote_auth_s3' does not exist in type 'DiscussionEmbedConfig'.  TS2769

    77 |               title: site?.project?.name,
    78 |               language: 'en_US',
  > 79 |               remote_auth_s3: 'll'
       |               ^
    80 |             }}
    81 |           />
    82 |         </div>
    ```

**Comment by jaequery** [comment]
also tried remoteAuthS3 but no luck.

**Comment by chrisjtang** [comment]
@jaequery Do you have an example repo that I can use to reproduce this issue?

**Comment by chrisjtang** [comment]
This issue is resolved by https://github.com/disqus/disqus-react/pull/136, which is now released and live in v1.1.4 here: https://www.npmjs.com/package/disqus-react.  We will publish an example repo in the coming days that is similar to https://disqus-sso-demo.glitch.me/.  Closing this issue as completed.

**Comment by chrisjtang** [comment]
Update: v1.1.5 also adds an interface that should fix most, if not all, typing error messages related to TS when using the DiscussionEmbed component for SSO.  And, to follow up on a previous comment, here are some example apps/repos for the SSO configuration of this package:

Javascript + React: [Disqus SSO React Demo](https://disqus-sso-react-demo.glitch.me/)

Typescript + React: [foodandstuff/foodandstuff-react at typescript · chrisjtang/foodandstuff](https://github.com/chrisjtang/foodandstuff/tree/typescript/foodandstuff-react)
